Representable/representable
# Multi export csv

Closes #515 

To test: 
1. go to a drive or organization page 
2. export communities in both geojson and csv (i changed both functions bit)
3. make sure both are correct and match up with the communities displayed on the webpage
